tests/filecheck/transforms/rocc-dedup.mlir

```diff
@@ -0,0 +1,60 @@
+// XFAIL: *
+// RUN: ./compiler/snax-opt %s -p convert-acc-to-csr | filecheck %s
+
+builtin.module {
+
+  "acc2.accelerator"() <{
+      name            = @gemmini,
+      fields = { k_LOOP_WS_CONFIG_BOUNDS.rs1=9, k_LOOP_WS_CONFIG_ADDRS_AB.rs1=10,
+        k_LOOP_WS_CONFIG_ADDRS_DC.rs1=11,
+        k_LOOP_WS_CONFIG_STRIDES_AB.rs1=12,
+        k_LOOP_WS_CONFIG_STRIDES_DC.rs1=13,
+        k_LOOP_WS.rs1=8,
+        k_LOOP_WS_CONFIG_BOUNDS.rs2=9,
+        k_LOOP_WS_CONFIG_ADDRS_AB.rs2=10,
+        k_LOOP_WS_CONFIG_ADDRS_DC.rs2=11,
+        k_LOOP_WS_CONFIG_STRIDES_AB.rs2=12,
+        k_LOOP_WS_CONFIG_STRIDES_DC.rs2=13,
+        k_LOOP_WS.rs2=8
+        },
+      launch_addr     = 0x3c0,
+      barrier         = 0x3c3
+  }> : () -> ()
+
+
+  func.func public @test() {
+    %t = arith.constant 32 : i32
+    %9 = "acc2.setup"(%t,%t,%t,%t,%t,%t,%t,%t,%t,%t,%t,%t) <{"accelerator" = "gemmini", "operandSegmentSizes" = array<i32: 12, 0>, 
+    "param_names" = [ "k_LOOP_WS_CONFIG_BOUNDS.rs1",
+        "k_LOOP_WS_CONFIG_ADDRS_AB.rs1",
+        "k_LOOP_WS_CONFIG_ADDRS_DC.rs1",
+        "k_LOOP_WS_CONFIG_STRIDES_AB.rs1",
+        "k_LOOP_WS_CONFIG_STRIDES_DC.rs1",
+        "k_LOOP_WS.rs1",
+        "k_LOOP_WS_CONFIG_BOUNDS.rs2",
+        "k_LOOP_WS_CONFIG_ADDRS_AB.rs2",
+        "k_LOOP_WS_CONFIG_ADDRS_DC.rs2",
+        "k_LOOP_WS_CONFIG_STRIDES_AB.rs2",
+        "k_LOOP_WS_CONFIG_STRIDES_DC.rs2",
+        "k_LOOP_WS.rs2"
+        ]}> : (i32, i32, i32, i32, i32, i32, i32, i32, i32, i32, i32, i32) -> !acc2.state<"gemmini">
+
+    %10 = "acc2.launch"(%9) <{"accelerator" = "gemmini"}> : (!acc2.state<"gemmini">) -> !acc2.token<"gemmini">
```

Comment:
No, this one assumes a deduplicated input where certain values are removed, which makes it impossible to emit an op, since it requires both values to be set.